tools/rna_tools/coprarna/coprarna.xml

```diff
@@ -0,0 +1,123 @@
+    <param argument="--ntdown" type="integer" value="100" label="Nucleotides downstream" help="Number of nucleotides downstream of '--region' to parse for targeting (def:100)" optional="true"/>
+    <param argument="--rcsize" type="float" value="0.5" min="0" max="1" label="Fraction of putative target homologs" help="Minimum amount (%) of putative target homologs that need to be available for a target cluster to be considered in the CopraRNA1 part (see --cop1) of the prediction (def:0.5)"/>
+    <param argument="--cores" type="integer" value="1" label="CPU cores" help="Number of cores to use for parallel computation (def:1)" optional="true"/>
+    <param argument="--winsize" type="integer" value="150" label="Window size" help="IntaRNA target (--tAccW) window size parameter (def:150)" optional="true"/>
+    <param argument="--maxbpdist" type="integer" value="100" label="Maximum basepair distance" help="IntaRNA target (--tAccL) maximum base pair distance parameter (def:100)" optional="true"/>
+    <param name="cons" type="select" label="Controls consensus prediction" help="'0' for off, '1' for organism of interest based consensus, '2' for overall consensus based prediction">
+            <option value="0" selected="true">Off</option>
+            <option value="1">Organism of interest based consensus</option>
+            <option value="2">Overall consensus based prediction</option>
+    </param>
+    <param argument="--topcount" type="integer" value="200" label="Top predictions" help="Specifies the number of top predictions to return and use for the extended regions plots (def:200)" optional="true"/>
+    <param argument="--cop1" truevalue="--cop1" falsevalue="" checked="False" type="boolean" label="CopraRNA1 prediction" help="Switch for CopraRNA1 prediction (def:off)"/>
+    <param argument="--nooi" truevalue="--nooi" falsevalue="" checked="False" type="boolean" label="No focus on the organism of interest" help="If set then the CopraRNA2 prediction mode is set not to focus on the organism of interest (def:off)"/>
+    <param argument="--enrich" truevalue="--enrich" falsevalue="" checked="False" type="boolean" label="DAVID-WS functional enrichment" help="If set DAVID-WS functional enrichment is calculated with given number of top predictions (def:off)"/>
+    <param argument="--enrich" type="integer" value="0" label="DAVID-WS functional enrichment" help="If set DAVID-WS functional enrichment is calculated with given number of top predictions (def:off)" optional="true"/>
+    <param argument="--root" type="float" value="1" label="Root function" help="Specifies root function to apply to the weights (def:1)"/>
+  </inputs>
+  <outputs>
+    <data format="csv" name="result" from_work_dir="./CopraRNA_result.csv" label="CopraRNA result csv"/>
+    <data format="txt" name="options" from_work_dir="./CopraRNA_option_file.csv" label="CopraRNA option log"/>
```

Comment:
The log file should be optional or just put onto stdout. An extra element in the Galaxy is probably not needed for the normal biologist.